docs/src/content/docs/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: CSS & Styling
```

Comment:
Wondering if it was intentional to name the file "CSS and Tailwind" but keep the title as "CSS & Styling"

Reply:
It kind of was, but maybe not a good decision? Perhaps I should stick to matching the title and rename it `css-and-styling.mdx`.